ReadMe

```diff
@@ -0,0 +1,47 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-HTML and CSS Basic
+
+This project is my first project as a Micronaut
+
+## Built With
+
+- Html
+- Css
+
+
+### Prerequisites
+ Basic knowledge in HTML and CSS
+
+
+### Deployment
+
+Using an IDE to develop html and css code to modify it or run it directly with your browser 
+
+## Authors
```

Comment:
- Provide information about how to dowload your project, install it locally and how to run your project.